cobalt/browser/switches.cc

```diff
@@ -147,6 +147,10 @@ const char kRequireCSPHelp[] =
     "Forbid Cobalt to start without receiving csp headers which is enabled by "
     "default in production.";
 
+const char kIgnoreCorsErrors[] = "ignore_cors";
```

Comment:
I'm not completely happy with the naming, because technically it's SOP that disallows anything with a different origin, and it's CORS ("Cross Origin Resource Sharing") that allows certain cross-origin connections that SOP blocks.

So it's semantically more correct to name it something like "cors_allow_always" or "allow-all-cross-origin", or mimicking the CORS header naming "access_control_allow_all".

-https://www.w3.org/TR/2020/SPSD-cors-20200602/#access-control-allow-origin-response-header